internal/driver/webhtml.go

```diff
@@ -53,90 +57,187 @@ button {
   margin-top: 5px;
   margin-bottom: 5px;
 }
-#reset {
-  margin-left: 10px;
-}
 #detailtext {
   display: none;
-  position: absolute;
+  position: fixed;
+  top: 20px;
+  right: 10px;
   background-color: #ffffff;
   min-width: 160px;
-  border-top: 1px solid black;
-  box-shadow: 2px 2px 2px 0px #aaa;
+  border: 1px solid #888;
+  box-shadow: 4px 4px 4px 0px rgba(0,0,0,0.2);
   z-index: 1;
 }
-#actionbox {
-  display: none;
-  position: fixed;
-  background-color: #ffffff;
-  border: 1px solid black;
-  box-shadow: 2px 2px 2px 0px #aaa;
-  top: 20px;
-  right: 20px;
-  z-index: 1;
+#closedetails {
+  display: inline;
```

Comment:
This is for the close button on the details pop up which I do want to flow independently of any other info in the pop up (it floats at the top-right corner of the pop-up).  The "float" seems pretty natural here (also no clearing is being done here)?

Not sure how flex would work to achieve this, but if you think flex would be better and can give me a pointer, I can try to incorporate that.